stdlib/3/_winapi.pyi

```diff
@@ -46,11 +52,10 @@ WAIT_TIMEOUT: int
 
 def CloseHandle(handle: int) -> None: ...
 
-# TODO: once literal types are supported, overload with Literal[True/False]
 @overload
-def ConnectNamedPipe(handle: int, overlapped: Union[int, bool]) -> Any: ...
+def ConnectNamedPipe(handle: int, overlapped: Literal[True]) -> Overlapped: ...
 @overload
-def ConnectNamedPipe(handle: int) -> None: ...
+def ConnectNamedPipe(handle: int, overlapped: Literal[False] = ...) -> None: ...
```

Comment:
Also need an overload for just `bool`

Reply:
For some reason I was assuming, type checkers would infer a proper union type. Seems I was wrong.